python/ray/includes/libcoreworker.pxd

```diff
@@ -161,6 +161,7 @@ cdef extern from "ray/core_worker/core_worker.h" nogil:
         ActorHandleSharedPtr GetActorHandle(const CActorID &actor_id) const
         pair[ActorHandleSharedPtr, CRayStatus] GetNamedActorHandle(
             const c_string &name)
+        CRayStatus GetAllActorNames(c_vector[c_string] *actor_names)
```

Comment:
All other `Getxxxx` requests return values in `return` type instead of `reference parameter`, can we make this consistent with them?

Reply:
done